database/Dockerfile

```diff
@@ -2,8 +2,13 @@ FROM registry.redhat.io/rhel8/postgresql-12
 
 # Install pg_repack
 USER root
-RUN curl http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/pg_repack-1.4.6-3.module_el8.5.0+842+bf20b04a.x86_64.rpm -o /tmp/pg_repack.rpm && \
-    rpm -ivh /tmp/pg_repack.rpm
+ADD /database/RPM-GPG-KEY-CENTOS /etc/pki/rpm-gpg/
```

Comment:
This is a Red Hat image `FROM registry.redhat.io/rhel8/postgresql-12`, I am not sure whether there's Centos GPG key present on Red Hat image.
Can you please check it?